components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineView.kt

```diff
@@ -25,7 +28,8 @@ import org.mozilla.geckoview.GeckoSession
 class GeckoEngineView @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
-    defStyleAttr: Int = 0
+    defStyleAttr: Int = 0,
+    preferredColorScheme: PreferredColorScheme? = null
```

Comment:
We usually try not to add to view constructors because we still want to be able to easily use views in layout XMLs. Let's expose a method instead to set the `preferredColorScheme`?